### DISCUSSION TITLE
Please upgrade the version of Uno.sqlite-wasm

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/uno#14844] [Q&A]
I'm having a sqlite issue after upgrading net8.0 on webassembly! Thanks.
@jeromelaban

**Comment by limeniye** [comment]
Hello, @Daoting. 
To help you, please attach the issue.

Currently, It looks like the following reply:
https://github.com/unoplatform/uno/discussions/12926#discussioncomment-6502893

**Comment by Daoting** [comment]
Update `Uno.sqlite-wasm3.40.0.2`, it's great! Thanks
